FinSearchUnified/Tests/Bundle/SearchBundleFindologic/QueryBuilder/QueryBuilderFactoryTest.php

```diff
@@ -98,10 +103,16 @@ public function testCreateQueryWithoutConditions()
      */
     public function testCreateQueryWithConditions()
     {
+        StaticHelperTest::createTestManufacturer([
+            'id' => 8,
+            'name' => 'FindologicVendor8'
+        ]);
+
         $criteria = new Criteria();
         $criteria->addCondition(new CategoryCondition([5, 12]));
+        $criteria->addCondition(new ManufacturerCondition([8]));
         $criteria->addCondition(new PriceCondition(1, 20));
-        $criteria->addCondition(new ProductAttributeCondition('vendor', '=', 'Findologic Rockers'));
+        $criteria->addCondition(new ProductAttributeCondition('findologic-vendor', '=', 'Findologic Rockers'));
```

Comment:
Why are you using `findologic-vendor`?

Reply:
Can you please click-test, if the manufacturer filter is returned on manufacturer pages? (in case it is multi-select)

We either don't show the filter, or it works with the manufacturer being pre-selected.

Reply:
> Can you please click-test, if the manufacturer filter is returned on manufacturer pages? (in case it is multi-select)
> 
> We either don't show the filter, or it works with the manufacturer being pre-selected.

Tobias we don't show the manufacturer filter on the manufacturer pages. If I understand correctly...